### PR TITLE
Add Fish Fillets NG

### PR DESCRIPTION
**Pull request by nextghost**
Hello, the game list is missing Fish Fillets NG, an old Czech puzzle game commercially released in 1997 and open-sourced in 2004.

### CODE REVIEW

File: games/f.yaml [hunk 1]
```diff
@@ -176,6 +176,30 @@
...
 
+- name: Fish Fillets NG
+  originals:
+  - Fish Fillets
+  type: remake
+  repo: 'https://sourceforge.net/projects/fillets/'
+  url: 'http://fillets.sourceforge.net/'
+  development: complete
+  status: playable
+  content: free
```
Comment (cxong) [inline games/f.yaml:187]:
```suggestion
  content: open
```
I read that the content is also GPL; note OSGC uses "free" as in "beer"

File: games/f.yaml [hunk 2]
```diff
@@ -176,6 +176,30 @@
...
+  - Lua
+  framework:
+  - SDL
+  license:
+  - GPL2
+  info: Cross-platform port of the original game sources.
+  updated: 2022-05-05
+  images:
+  - 'http://fillets.sourceforge.net/img/screenshot/ffng-pyramid.png'
+  - 'http://fillets.sourceforge.net/img/screenshot/ffng-chest.png'
```
Comment (cxong) [inline games/f.yaml:199]: images should be HTTPS otherwise they don't show when embedded

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/f.yaml (hunk 1 above)
Comment (nextghost) [reply games/f.yaml]: Thanks, fixed.

File: games/f.yaml (hunk 2 above)
Comment (nextghost) [reply games/f.yaml]: Unfortunately, the SourceForge project site does not support HTTPS so I hade to replace both images with one from Wikipedia.

**Review by cxong — APPROVED** [review]

**Pull request merged**